--- ./usr/local/bin/secpanel.org	2008-03-14 00:46:25.201669527 +0100
+++ ./usr/local/bin/secpanel	2008-03-14 00:47:02.600124055 +0100
@@ -1297,7 +1297,7 @@ proc connect {{cm ""}} {
 
 	set cfr "$cf-run"
 	set crfile [open "$cfr" w]
-	set runstring "exec $configs(xtermbin) $icontag $titlepar \"SSH Connection - $title\" \
+	set runstring "exec $configs(xtermbin) $icontag $titlepar \"SSH - $title\" \
 	    -e $quotepar $waittag $woption $cf $woption $quotepar"
 	puts $crfile $runstring
 	close $crfile
