--- secpanel-0.6.0/usr/local/bin/secpanel~	2010-02-19 13:44:06.000000000 +0100
+++ secpanel-0.6.0/usr/local/bin/secpanel	2010-02-22 16:37:12.112675822 +0100
@@ -29,8 +29,9 @@
 
 set spversion 0.6.0
 
-set libdir "[file dirname [info script]]/../lib/secpanel"
+# set libdir "[file dirname [info script]]/../lib/secpanel"
 # set libdir "$env(HOME)/secpanel_devel/secpanel-0x/lib"
+set libdir "/usr/share/secpanel"
 set sshdir "$env(HOME)/.ssh"
 
 global actdirsel;
