--- secpanel-0.6.0/usr/local/bin/secpanel~	2010-02-22 16:37:50.420179394 +0100
+++ secpanel-0.6.0/usr/local/bin/secpanel	2010-02-22 16:39:30.760145987 +0100
@@ -1403,7 +1403,7 @@
 			$fork_tag $gateway_tag $compress_tag $compressval_tag \
 			$lf_tag $rf_tag $host $command_tag"
 
-		set runstring "exec $configs(xtermbin) $icontag $titlepar \"SSH Connection - $title\" \
+		set runstring "exec $configs(xtermbin) $icontag $titlepar \"SSH - $title\" \
 			$execpar $quotepar $waittag $woption <CONNFILE> $woption $quotepar"
 
 		provrunfile term bg $connstring $runstring
