--- secpanel-0.6.0/usr/local/lib/secpanel/spdistkey~	2010-02-19 11:50:57.000000000 +0100
+++ secpanel-0.6.0/usr/local/lib/secpanel/spdistkey	2010-02-22 16:47:16.979259946 +0100
@@ -8,8 +8,14 @@
 # Author: Steffen Leich-Nienhaus <steffen.leich _at_ gmail.com>
 ##########################################################################
 
+function do_exit {
+       echo
+       echo -e "Key-Distribution finished\nPress <Return> to continue"
+       read
+       exit
+}
 
-if [ -z $4 ]
+if [ -z $5 ]
 then
 cat <<EOF
 
@@ -59,7 +65,7 @@
 then
     echo
     echo "There was an error connecting to the remote site"
-    echo -e "Parameters:\n\tHost:\t$HOST\n\tUser:\t$USER"
+    echo -e "Parameters:\n\tHost:\t$HOST:$PORT\n\tUser:\t$USER"
     echo
     echo "Canceling the key-transfer"
     do_exit
@@ -68,6 +74,6 @@
     echo "    The key could not be found on this host"
     echo "    -> Transfering your public key to remote host"
     echo
-    $SSHBIN -l $USER $HOST "cat >> \$HOME/.ssh/authorized_keys; chmod 600 \$HOME/.ssh/authorized_keys; chmod 700 \$HOME/.ssh" < $IDENTITY
+    $SSHBIN -l $USER -p $PORT $HOST "cat >> \$HOME/.ssh/authorized_keys; chmod 600 \$HOME/.ssh/authorized_keys; chmod 700 \$HOME/.ssh" < $IDENTITY
 fi
 
