--- ./usr/local/bin/secpanel.org	2009-04-10 10:09:06.363971463 +0200
+++ ./usr/local/bin/secpanel	2009-04-10 10:11:13.963967397 +0200
@@ -1420,7 +1420,7 @@ proc connect {{cm ""}} {
 	    $fork_tag $gateway_tag $compress_tag $compressval_tag \
 	    $lf_tag $rf_tag $host $command_tag"
 
-	set runstring "exec $configs(xtermbin) $icontag $titlepar \"SSH Connection - $title\" \
+	set runstring "exec $configs(xtermbin) $icontag $titlepar \"SSH - $title\" \
 	    $execpar $quotepar $waittag $woption <CONNFILE> $woption $quotepar"
 
 	provrunfile term bg $connstring $runstring
